### ISSUE TITLE
Required section property casing

### ISSUE THREAD

**Issue by MrWolfPST** [Azure/azure-functions-openapi-extension#227]
I have some properties decorated with JsonRequired attribute and JsonProperty attribute with custom property name specified.
In the properties section of swagger.json, property name respect casing of property name value of JSonProperty attribute, but in the required section the name is with first letter lower cased.

**Comment by choipureum** [comment]
I think it's related to #83

**Comment by MrWolfPST** [comment]
Yes, it can be correlated. My problem is on required section only, but I use JsonProperty with custom property name, that is the workaround suggested in the issue #83.

**Comment by justinyoo** [comment]
As a default, `camelCasing` is applied to all unless it's specified through `JsonProperty(...)`. It's because of this line:

https://github.com/Azure/azure-functions-openapi-extension/blob/c5d78bc24ecc626e96c3d3ab17fcfcc9c0c62a24/src/Microsoft.Azure.WebJobs.Extensions.OpenApi/OpenApiHttpTriggerContext.cs#L112

and here:

https://github.com/Azure/azure-functions-openapi-extension/blob/c5d78bc24ecc626e96c3d3ab17fcfcc9c0c62a24/src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/OpenApiHttpTriggerContext.cs#L112

Let me take a look how I can configure the default behaviour.

**Comment by MrWolfPST** [comment]
As workaroud I added Required = Required.Always on JsonPropertyAttributes that have PropertyName specified
